src/diffusers/models/samplers/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .downsamplers import Downsample1D, Downsample2D, FirDownsample2D, KDownsample2D
```

Comment:
I don't think "samplers" is a good name as many people understand a "sampler" as a "scheduler". Let's just keep the files on the src/diffusers/models level for the moment and just call them downsampling.py and upsampling.py

Reply:
How about `interpolation`?

Reply:
We could also do `blocks`? I think that's more generic. I'd really like to move these files to a separate module and keep `models` as lean as possible.